src/features/navigation/RootNavigator/routes.ts

```diff
@@ -447,6 +448,13 @@ export const routes: Route[] = [
     options: { title: t`Vérification d'identité` },
     secure: true,
   },
+  {
+    name: 'IdentityCheckStatus',
+    component: IdentityCheckStatus,
+    path: 'choix-du-status',
```

Comment:
```suggestion
    path: 'verification-identite/profil/statut',
```

à voir avec @BenStirrup

Reply:
Ouais j'ai fait n'importe quoi la ! ahah